### PR TITLE
COMMERCE-10627 fixing the ordering script to fix the issue with filte…

### PR DESCRIPTION
**Pull request by MahmoudAzzam**
JIRA : https://issues.liferay.com/browse/COMMERCE-10627

### CODE REVIEW

**Comment by MahmoudAzzam** [comment]
ci:test:sf

**Comment by MahmoudAzzam** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_publisher/configuration/ordering.jsp [hunk 1]
```diff
@@ -102,23 +102,28 @@ CPPublisherConfigurationDisplayContext cpPublisherConfigurationDisplayContext =
...
-			var orderByTypeField = orderByTypeContainer.one('.order-by-type-field');
-
-			var newVal = orderByTypeField.val() === 'ASC' ? 'DESC' : 'ASC';
-
-			orderByTypeField.val(newVal);
-		},
-		'.icon'
-	);
+	var order = A.one('#<portlet:namespace />ordering');
+	if (order) {
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_publisher/configuration/ordering.jsp:106]: Ehi @MahmoudAzzam I don't think this is the right fix. What I understand from the previous version of the table (the one with aui:row) this was always working while this is occasionally working. 
I think you should focus on the generated id of the row, most probably the new tag is not prefixing the id value with the <portlet:namespace />...

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]
Please find a better solution

**Comment by MahmoudAzzam** [comment]
ci:test:sf

**Comment by MahmoudAzzam** [comment]
ci:test:relevant

**Review by andrea-ale-sbarra — APPROVED** [review]

**Comment by MahmoudAzzam** [comment]
ci:test:sf

**Review by andrea-ale-sbarra — APPROVED** [review]
thank you @MahmoudAzzam

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Pull request closed without merging**